test/go-report-card-test/Dockerfile

```diff
@@ -1,7 +1,8 @@
-FROM golang:1
+FROM golang:1.16
 
 WORKDIR /app
 
+ARG GO111MODULE=auto
```

Comment:
This is a bit of a hack, because it doesn't address `go get` being deprecated, and it will also modify go.mod and go.sum if a humans runs it locally. I'm still trying to figure out how to get line 7 to work (`make install` inside the goreportcard repo after download) with `go install` instead of `go get`. But in the mean time, better to have a working build than to have this 100% perfect.

Reply:
Let's just remove `go-report-card-test` altogether as I don't think we get much value out of it anymore. NTH has been without it for awhile and I'll submit the PR to remove for AEMM shortly